v4/inprocess-proxy/README.md

```diff
@@ -0,0 +1,56 @@
+# Example app that creates two proxies in process
+
+## To start
+
+### Configure database
+```sql
+CREATE USER unleash_user WITH PASSWORD 'passord';
+CREATE DATABASE unleashwithproxy WITH OWNER 'unleash_user';
+```
+
+### Install dependencies
+```shell
+$ yarn
```

Comment:
If we're gonna use `$` to indicate that you're running a command, we should be consistent. This seems to be the only place where we do that in this doc, so I vote we take it out.

Of course, I may be misunderstanding what's happening here! Feel free to let me know and to disagree.

```suggestion
yarn
```